README.md

```diff
@@ -20,18 +20,24 @@ $ sudo ./install.sh
 ```
 
 ### Installed Tools
----
-* cyberway.eosio-cpp
-* cyberway.eosio-cc
-* cyberway.eosio-ld
-* cyberway.eosio-init
-* cyberway.eosio-abidiff
-* cyberway.eosio-wasm2wast
-* cyberway.eosio-wast2wasm
-* cyberway.eosio-ranlib
-* cyberway.eosio-ar
-* cyberway.eosio-objdump
-* cyberway.eosio-readelf
+
+ * cyberway-abidiff
+ * cyberway-cpp
+ * eosio-abigen
+ * eosio-ar
+ * eosio-cc
+ * eosio-init
+ * eosio-ld
+ * eosio-nm
+ * eosio-objcopy
+ * eosio-objdump
+ * eosio-pp
+ * eosio-ranlib
+ * eosio-readelf
+ * eosio-readobj
+ * eosio-strip
+ * eosio-wasm2wast
+ * eosio-wast2wasm
```

Comment:
@boostmultifruit are right. Installed only next utilities (fresh install):

- cyberway-abidiff
- cyberway-cpp
- eosio-cc
- eosio-init
- eosio-ld
- eosio-wasm2wast
- eosio-wast2wasm
- llvm-ranlib
- llvm-ar
- llvm-objdump
- llvm-readelf

And next which not specified in EOS readme file:
- llvm-nm
- llvm-objcopy
- llvm-readobj
- llvm-strip
- eosio-abigen
- eosio-pp

Oh my god! What's the strange names for utilities?! I would prefer that all utilities have one `cyberway-` prefix. So it will be easier to use this set when cross-compiling. But any way it's all for another issue...

Reply:
it's easy to remove mentioned 6 names from README . But what about the rest of the names? 
A month ago there was a decision to add the "cyberway" prefix to all names. Since sources were not fixed, it was decided to remove this prefix back. Now, making changes to sources is very risky.